fv3core/utils/grid.py

```diff
@@ -476,8 +476,8 @@ def jsc(self):
 
     @property
     def jec(self):
-        """end of the compute domain along the y-axis"""
-        return self.origin[1] + self.domain[1]
+        """last index of the compute domain along the y-axis"""
+        return self.origin[1] + self.domain[1] - 1
```

Comment:
you indeed want to do this to ied, jed as well. self.origin[1] + self.domain[1] is equivalent to grid.nic, while subtracting 1 gets you the index of the top value. Didn't notice this in the first review.

Reply:
Updated.

Reply:
Thanks!